network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreExtensionsIT.java

```diff
@@ -506,26 +506,39 @@ public void modifyExtensionsNetwork() {
             ConnectablePosition connectablePositionload = load.getExtension(ConnectablePosition.class);
             assertNotNull(connectablePositionload);
             connectablePositionload.getFeeder().setName("v1loadModified").setOrder(10).setDirection(ConnectablePosition.Direction.BOTTOM);
+            ConnectablePosition connectablePositionTwt2 = twt2.getExtension(ConnectablePosition.class);
+            assertNotNull(connectablePositionTwt2);
+            connectablePositionTwt2.getFeeder1().setName("TWT2Modified.1").setOrder(20).setDirection(ConnectablePosition.Direction.BOTTOM);
+            connectablePositionTwt2.getFeeder2().setName("TWT2Modified.2").setOrder(20).setDirection(ConnectablePosition.Direction.BOTTOM);
+            ConnectablePosition connectablePositionTwt3 = twt3.getExtension(ConnectablePosition.class);
+            assertNotNull(connectablePositionTwt3);
+            connectablePositionTwt3.getFeeder1().setName("TWT3Modified.1").setOrder(30).setDirection(ConnectablePosition.Direction.BOTTOM);
+            connectablePositionTwt3.getFeeder2().setName("TWT3Modified.2").setOrder(30).setDirection(ConnectablePosition.Direction.BOTTOM);
+            connectablePositionTwt3.getFeeder3().setName("TWT3Modified.3").setOrder(30).setDirection(ConnectablePosition.Direction.BOTTOM);
+            service.flush(readNetwork);
```

Comment:
End the try block just after flush, and begin a new try block with the creation of a new NetworkStoreService object :
To be consistent with what is done in the other methods (cgmesExtensionsTest for example)